src/QATeamLead.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Scanner;
+
+public class QATeamLead extends QAEmployee {
+	private int qaExperience;
```

Comment:
add some validation here

Reply:
I added the validation for this attribute.